src/vmm/src/devices/virtio/queue.rs

```diff
@@ -222,6 +223,50 @@ pub struct Queue {
     /// Guest physical address of the used ring
     pub used_ring_address: GuestAddress,
 
+    /// This is a user space pointer
+    /// to the available ring, set up by the
+    /// guest. Getting access to the underling
+    /// data structure should only occur after the
```

Comment:
I am not sure there is ambiguity. This is a 'pointer'. It would be strange for it to be a guest pointer. I will update this to `host pointer`.

Reply:
The terminology "host virtual address", "guest virtual address", "guest physical address" is very common in the virtualization world. It would be nice if we were accurate.

Reply:
ok

Reply:
done